src/travel-details-map-screen/TravelDetailsMapScreenComponent.tsx

```diff
@@ -43,11 +46,18 @@ export const TravelDetailsMapScreenComponent = ({
   const {location: geolocation} = useGeolocationState();
 
   const features = useMemo(() => createMapLines(legs), [legs]);
-  const bounds = getMapBounds(features);
+  const bounds = !initialVehiclePosition ? getMapBounds(features) : undefined;
+  const centerPosition = initialVehiclePosition?.location
+    ? [
+        initialVehiclePosition?.location?.longitude,
+        initialVehiclePosition?.location?.latitude,
+      ]
+    : undefined;
 
   const {t} = useTranslation();
   const controlStyles = useControlPositionsStyle();
   const vehicles = useGetLiveServiceJourneyVehicles(initialVehiclePosition);
```

Comment:
The parameter name `initialVehiclePosition` was sort of confusing to me, as I thought it was actually just the vehicle position, and I couldn't understand how that could be the input for `useGetLiveServiceJourneyVehicles`. I think it would be quite clearer to name the type and param something like `VehicleWithPosition`.